### ISSUE TITLE
Rolling: increase Taskbar height to 55 pixels

### ISSUE THREAD

**Issue by star-buck** [blue-systems/netrunner-issues-old#182]

**Comment by NuLogicSystems** [comment]
This line in the theme packages defaultpanel layout.js file, is overriding anything preset in the diversions plasmashellrc file: panel.height = screenGeometry(panel.screen).height > 1024 ? 38 : 27

When the theme is applied, it creates a new panel using this setting, effectively setting the new panel to 27. Can we have shadeslayer or leszek take a look at it?

**Comment by llelectronics** [comment]
Not sure how Rolling sets the stuff up for the desktop. On Debian we use LNF packages which set the Plasma theme and configuration settings. 
So we have something like this https://github.com/netrunner-artwork/artwork-lnf-theme-netrunner-desktop/blob/master/usr/share/plasma/look-and-feel/org.kde.netrunner-desktop.desktop/contents/layouts/org.kde.plasma.desktop.defaultPanel/contents/layout.js#L28
in our LNF package. 
If you create your own LNF package for Rolling you could just edit this line and set the panel to 55 pixels.

**Comment by star-buck** [comment]
Sounds good to use the Netrunner Black LNF and adjust as Netrunner Rolling
Black LNF or so... Although ultimately it should just differ for wallpaper
:)

**Comment by llelectronics** [comment]
For the future if Rolling changes more of the artwork(icons, plasma, widget, colors) it makes sense to ship its own LNF theme imho.

**Comment by NuLogicSystems** [comment]
If this is the case, then why do we ship a plasmashellrc file in the default-settings-plasma package in the first place?
I already do ship my own LNF themes, that is not the issue. The issue is that I do not understand that line, so I do not know what needs to be changed.

**Comment by llelectronics** [comment]
Ah I see. Yeah this is a left over from an older release where we did not use the LNF for setting the height. 
It needs cleanup. Sorry for the confusion. Though it might be used by non LNF themes.

**Comment by NuLogicSystems** [comment]
Thanks for the explanation, but I still do not know what to change this lines numbers to?
Should I just make them both 55?
Do I need to change the 1024 to something higher?

**Comment by llelectronics** [comment]
Leave the plasmashellrc as is and only set the 55 in the LNF for resolutions higher than 1024. I don't think it looks great on smaller screens.

**Comment by NuLogicSystems** [comment]
So I just change the last number then: 27 - 55?
The reason I ask is, I've noticed that the first number seems to only change the 1024 size.
It still changes the plasmashellrc file in the home directory to the following for all other resolutions:
```
[plasmaviews][panel 2] [horizontal1366]
thickness=27
```

**Comment by llelectronics** [comment]
Ah sorry. Its checking for height greater than 1024 not width. 
So change the first (the bigger number) in the LNF I would say.

**Comment by NuLogicSystems** [comment]
But wouldn't that still set 1366x720 to 27?
so would I be better off with 55 : 38?

**Comment by llelectronics** [comment]
> But wouldn't that still set 1366x720 to 27?

Yeah. Exactly. 
> so would I be better off with 55 : 38?

Maybe. Needs testing I think to get it looking good or the same with the different resolutions.

**Comment by NuLogicSystems** [comment]
Or the subsystem need to be changed to work with the newer standard (widthxhight).
I mean, when was the last time you saw any non-widescreen monitors/televisions (hightxwidth)?
That is besides in a pawn shop, or antique store, etc.

**Comment by NuLogicSystems** [comment]
OK, 45 looks better (about the same) at 1366x720 as 55 at 1920x1024.

I  think 55:45 should take care of most current common screen resolutions on the market today.
I'm testing this out now.

**Comment by NuLogicSystems** [comment]
Actually. I probably should ask this first.
@star-buck 
What resolution was your monitor when you decided on 55 pixels?

**Comment by star-buck** [comment]
this works great in live mode, but after install, it falls back to small as before:
![Screenshot_20190409_130444](https://user-images.githubusercontent.com/840070/55795688-adf53c80-5ac8-11e9-9a39-6d0462e42fd1.png)

**Comment by NuLogicSystems** [comment]
That's strange, this setting is resolution dependent and set by theme.
Was the resolution different in live mode than on installed system?

**Comment by star-buck** [comment]
I cant imagine why, but will test again next time for sure.

**Comment by NuLogicSystems** [comment]
I just checked my new install, and both live and installed was set to 44.
However, that was expected since it's resolution is below the 1024 height (1366x768).

**Comment by NuLogicSystems** [comment]
@star-buck,
What is your monitor resolution?

**Comment by star-buck** [comment]
1680x1050...

**Comment by llelectronics** [comment]
If the LNF is not correctly loaded by default this could explain this problem here aswell.

**Comment by NuLogicSystems** [comment]
The lnf is being parsed, I just checked on one of my other test systems (1920x1080), and I got 55px after install.  Adding another default panel also parsed the lnf and was 55px as well.

However, this would only apply to the black lnf as i did not change the netrunner rolling lnf, nore did I change the plasmashellrc file. The later only sets these two resolutions (1366 and 1920) to 34px and only for panel 1, which makes no matter as the lnf will write to the users plasmashellrc file as panel 2, and so on. Changing the lnf and clicking to use it's desktop layout will create another panel # and use it's default, or the systems default if none exist in the theme.
